### PR TITLE
Update with SME comments

### PR DESCRIPTION
**Pull request by ReeceNana**
#12

### CODE REVIEW

File: README.adoc [hunk 1]
```diff
@@ -67,27 +69,23 @@ include::{common-includes}/gitclone.adoc[]
...
-  -p 4318:4318 \
-  -p 14250:14250 \
   -p 14268:14268 \
-  -p 14269:14269 \
+  -p 14250:14250 \
   -p 9411:9411 \
   jaegertracing/all-in-one:1.46
 ```
 
 You can find information about the Jaeger server and instructions for starting the all-in-one executable file in the [Jaeger documentation](https://www.jaegertracing.io/docs/1.22/getting-started/#all-in-one).
```
Comment (Azquelt) [inline README.adoc]: Update the version in the docs link (1.22 -> 1.46)

File: README.adoc [hunk 2]
```diff
@@ -415,13 +415,13 @@ To access the Tracer, inject it into a bean by using the [hotspot=inject file=0]
...
 
-After starting the span, we make it current with the makeCurrent() call. Making a span current means that any new spans created in the same thread (either automatically by open liberty, or manually by calling the API) will use this span as their parent span.
+After starting the span, you make it current with the makeCurrent() call. Making a span current means that any new spans created in the same thread (either automatically by open liberty, or manually by calling the API) will use this span as their parent span.
 
-When we call makeCurrent() , it returns a Scope object. We need to call Scope.close() when we want the span to stop being the current span. The easiest way to do this is to call makeCurrent() in a try-with-resources block, which ensure the Scope is automatically closed at the end of the block.
+When calling makeCurrent() , it returns a Scope object. You need to call Scope.close() when you want the span to stop being the current span. The easiest way to do this is to call makeCurrent() in a try-with-resources block, which ensure the Scope is automatically closed at the end of the block.
```
Comment (Azquelt) [inline README.adoc]: > You need to call Scope.close() when you want the span to stop being the current span.

This sounds like you only need to call `Scope.close()` **if** you want the span to stop being the current span.

However, users **must always** call `Scope.close()`.

File: README.adoc [hunk 3]
```diff
@@ -67,27 +69,23 @@ include::{common-includes}/gitclone.adoc[]
...
   -p 16686:16686 \
-  -p 4317:4317 \
-  -p 4318:4318 \
-  -p 14250:14250 \
   -p 14268:14268 \
-  -p 14269:14269 \
+  -p 14250:14250 \
   -p 9411:9411 \
   jaegertracing/all-in-one:1.46
 ```
```
Comment (Azquelt) [inline README.adoc]: Why the change here? This doesn't match the [Jaeger documentation for 1.46](https://www.jaegertracing.io/docs/1.46/getting-started/#all-in-one).

In particular, the removal of port 4317 will prevent spans being delivered to Jaeger.

File: README.adoc [hunk 4]
```diff
@@ -486,56 +411,32 @@ InventoryResource.java
 include::finish/inventory/src/main/java/io/openliberty/guides/inventory/InventoryResource.java[]
 ----
 
-In order to access the Tracer, the [hotspot=inject file=0]`@Inject` annotation from the Contexts and Dependency Injections API injects the Tracer into a bean. 
+To access the Tracer, inject it into a bean by using the [hotspot=inject file=0]`@Inject` annotation from the Contexts and Dependency Injections API.
 
-Before the [hotspot=manager hotspot=getSystem file=0]`InventoryManager` calls the `system` service, the `spanBuilder()` and `startSpan()` Tracer APIs create and start a span called [hotspot=getPropertiesSpan file=0]`GettingProperties`.
+Before the [hotspot=manager hotspot=getSystem file=0]`InventoryManager` to call the `system` service, create and start a span called [hotspot=getPropertiesSpan file=0]`GettingProperties` by using the `spanBuilder()` and `startSpan()` Tracer APIs.
```
Comment (Azquelt) [inline README.adoc]: Sentence needs rewritten, the grammar is all wrong.

Maybe to this?

Before the `InventoryManager` calls the `system` service, it creates and starts a span called `GettingProperties` by using the `spanBuilder()` and `startSpan()` Tracer APIs.

File: README.adoc [hunk 5]
```diff
@@ -486,56 +411,32 @@ InventoryResource.java
...
 
-The [hotspot=try file=0]`try` block is called a `try-with-resources` statement that the [hotspot=scope file=0]`scope` object is closed at the end of the statement. It’s good practice to define custom spans inside such statements. Otherwise, any exceptions that are thrown before the span closes will leak the active span.
+When starting a span, you will need to end it by calling end() on the span. If you don't end a span, it won't be recorded at all and won't show up in Jaeger. This code ensures that end() is always called by including it in a finally block.
```
Comment (Azquelt) [inline README.adoc]: Add code formatting to `end()`

File: README.adoc [hunk 6]
```diff
@@ -486,56 +411,32 @@ InventoryResource.java
...
+The [hotspot=try file=0]`try` block is called a `try-with-resources` statement that the [hotspot=scope file=0]`scope` object is closed at the end of the statement. It’s good practice to define custom spans inside such statements. Otherwise, any exceptions that are thrown before the span closes will leak the active span.
 
+After starting the span, you make it current with the makeCurrent() call. Making a span current means that any new spans created in the same thread (either automatically by open liberty, or manually by calling the API) will use this span as their parent span.
 
-ifndef::cloud-hosted[]
-To check out the traces that contain the `GettingProperties` span, visit the http://localhost:9081/inventory/systems/localhost URL and then your Jaeger server at the http://localhost:16686 URL.
-endif::[]
+When calling makeCurrent() , it returns a Scope object. You must always call Scope.close() in order for the span to stop being the current span. The easiest way to do this is to call makeCurrent() in a try-with-resources block, which ensures the Scope is automatically closed at the end of the block.
```
Comment (Azquelt) [inline README.adoc]: This doesn't read well. The first paragraph is muddled and talks about defining a custom span which isn't what this block does.

How does this look instead?

```
After starting the span, you make it current with the `makeCurrent()` call. Making a span current means that any new spans created in the same thread (either automatically by open liberty, or manually by calling the API) will use this span as their parent span.

The `makeCurrent()` call returns a `Scope`. It's very important to always close the `Scope`, which stops the span being current and makes the previous span current again. The recommended way to do this is to use a [hotspot=try file=0] `try-with-resources` block, which makes sure that the `Scope` is automatically closed at the end of the block.
```

**Comment by ReeceNana** [comment]
SME comments moved to working branch - "sme-review-updates"  #21 

cc/ @gkwan-ibm

**Comment by gkwan-ibm** [comment]
close
- invalid branch
